certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+          "format": "int64",
+          "description": "Gets or Sets Id",
+          "type": "integer"
+        },
+        "role": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderPartyRole"
+        },
+        "roleType": {
+          "description": "Gets or Sets RoleType",
+          "type": "string"
+        },
+        "name": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderName"
+        },
+        "addresses": {
+          "description": "Gets or Sets Addresses",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.TitleOrderAddress"
+          }
+        },
+        "communications": {
+          "description": "Gets or Sets Communications",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Communication"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderNote": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "date": {
+          "description": "Gets or Sets Date",
+          "type": "object"
+        },
+        "publisher": {
+          "description": "Gets or Sets Publisher",
+          "type": "string"
+        },
+        "message": {
+          "description": "Gets or Sets Message",
+          "type": "string"
+        },
+        "title": {
+          "description": "Gets or Sets Title",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.Document": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "attributes": {
+          "description": "Gets or Sets Attributes",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "id": {
+          "description": "Gets or Sets Id",
+          "type": "integer"
+        },
+        "key": {
+          "description": "Unique document Id",
+          "type": "string"
+        },
+        "category": {
+          "description": "Gets or Sets Category",
+          "type": "string"
+        },
+        "fileName": {
+          "description": "Gets or Sets FileName",
+          "type": "string"
+        },
+        "description": {
+          "description": "Gets or Sets Description",
+          "type": "string"
+        },
+        "date": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.Date"
+        },
+        "publisher": {
+          "description": "The publishing author of the document",
+          "type": "string"
+        },
+        "submissionReference": {
+          "description": "Sender's reference key for the document",
+          "type": "string"
+        },
+        "documentBase64": {
+          "description": "Gets or Sets DocumentBase64",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrder": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.